src/WebJobs.Script.WebHost/Program.cs

```diff
@@ -73,6 +73,7 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args = null)
 
                     loggingBuilder.AddDefaultWebJobsFilters();
                     loggingBuilder.AddWebJobsSystem<WebHostSystemLoggerProvider>();
+                    loggingBuilder.AddConsoleIfEnabled(context);
```

Comment:
Please remove this - why do you keep adding it back?

Reply:
To test :( There is not file logger. This is the only logging for process startup for java worker